### PR TITLE
add bazl laermbelastungskataster layers

### PR DESCRIPTION
**Pull request by ltclm**

### CODE REVIEW

File: chsdi/locale/fi/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -1701,7 +1701,7 @@ msgid "ch.bazl"
 msgstr "UFAC"
 
 msgid "ch.bazl.heliports-gebirgslandeplaetze"
-msgstr "Heliports/Gebirgslandeplätze"
+msgstr "Heliports/pl. atterrament muntogna"
```
Comment (AFoletti) [inline chsdi/locale/fi/LC_MESSAGES/chsdi.po]: Uppercase

**Comment by AFoletti** [comment]
please rebase, and check the legend images for max 480px wide

**Comment by ltclm** [comment]
images have been resized to 480px max
```bash
for file in ls ch.bazl.laermbelastungskataster-*; do width=$(identify -format "%[fx:w]" $file 2>/dev/null); if [[ $width -gt 480 ]]; then echo "$file ..."; mv $file -> ${file%.*}_big.png; convert ${file%.*}_big.png -resize 480x $file; fi; done
```

**Comment by loicgasser** [comment]
LGTM, tests will carry on failing until model is merged.

**Pull request merged**